app/models/member.rb

```diff
@@ -56,6 +56,9 @@ class Member < ApplicationRecord
   before_validation :set_default_address_fields
   before_validation :downcase_email
 
+  after_save :send_welcome_text, if: -> {
+    reminders_via_text? && (saved_change_to_phone_number? || saved_change_to_reminders_via_text?)
```

Comment:
Will `saved_change_to_reminders_via_text?` return true if someone has text messages turned on but then disables them?

Reply:
Yep the first clause should ensure that uncheck and save does not text. There's a clause in the exercising test here: https://github.com/chicago-tool-library/circulate/pull/1407/files#diff-43b9c2473f1b9fb4712413041dbe1968d063150039a6d4b7b38810e5bcf880b6R224-R227

Reply:
Awesome. Ship it!